patterns/footer-1.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Title: WooCommerce Footer 1
+ * Slug: woocommerce-blocks/footer-1
+ * Categories: WooCommerce
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->
+<footer class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"var:preset|spacing|40","bottom":"0","left":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
```

Comment:
I think the footer should probably have full width: `<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->` and `<footer class="wp-block-group alignfull">` similar to the other footers we already have, like [footer-simple.php](https://github.com/woocommerce/woocommerce-blocks/blob/6fe5fbb887527242dab9ac7675ea2e54dc365dd8/patterns/footer-simple.php#L10).

Reply:
Addressed in 16b7481.